README.md

```diff
@@ -7,8 +7,8 @@ Amahi iOS App, new from scratch, in Swift.
 
 ## Requirements
 
-- iOS 9.0+
-- Xcode 9.0+
+- iOS 11.0+
+- Xcode 10.0+
```

Comment:
Why did you change this?

Reply:
I changed setter for `statusBarStyle` by overriding `preferredStatusBarStyle` since `statusBarStyle` was deprecated in previous version of ios. 
I thought doing that won't work in the old ios versions. It seems like I may have been wrong. I will revert to old previous commit. Pardon my mistake.

Reply:
Okay. Instead you can check for the specific version of ios and use the deprecated code on old version and the lastest on the new.

Something like this

```
if #available(iOS 9.0, *) {
    // use the feature only available in iOS 9
    // for ex. UIStackView
} 
else {
    // or use some work around
}
```

Reply:
Thanks for the tip.
I've used a different method since `#available` was giving me errors 

This is what I have come up with 
`        let systemVersion = UIDevice.current.systemVersion
        
        if((Double)(systemVersion) >= 10.0){
            override var preferredStatusBarStyle: UIStatusBarStyle { return .lightContent}
        } else {
            UIApplication.shared.statusBarStyle = .lightContent
        }`